src/main/java/igym/entities/User.java

```diff
@@ -13,24 +13,48 @@ public class User {
     private UUID id;
     private String name;
 
-    public User(UUID id, String name) {
-        this.id = id;
+    public User() {
+
+    }
```

Comment:
I think we must include a validation here. We can not accept a constructor without the `name` parameter.

But, if this is necessary, I suggest this change just to make our code cleaner.

```suggestion
    public User() {}
```

Reply:
The "alt" + "shift" + "F" let the constructor like this :( 
`public User() {
}`

Reply:
You are right @itaayp , the point is that SpringBoot needs an empty constructor to build it in the DB. So, yeah, I will accept your suggestion.